src/lib/support/SafeInt.h

```diff
@@ -35,7 +35,7 @@ namespace chip {
  * in the range of valid values for T.
  */
 template <typename T, typename U>
-bool CanCastTo(U arg)
+constexpr bool CanCastTo(U arg)
```

Comment:
Probably worth factoring this out as a simple one-liner PR.

Reply:
Yes. (The reason I changed this actually no longer exists in this PR. _Most_ uses of `CanCastTo` are testing dynamically computer values, but I think there are a small number of tests that could be replaced with `static_assert`.  `CanCastTo` couldn't be `constexpr` in C++11, but can be now in C++14.)